crates/bevy_ecs/src/storage/table.rs

```diff
@@ -666,11 +814,15 @@ impl Tables {
         self.tables.is_empty()
     }
 
+    /// Fetches a [`Table`] by its [`TableId`].
+    ///
+    /// Returns `None` if `id` is invalid.
     #[inline]
     pub fn get(&self, id: TableId) -> Option<&Table> {
         self.tables.get(id.index())
     }
 
+    /// Fetches mutable references to two different [`Table`]s.
```

Comment:
Something bad happens here if `a` and `b` are the same, right? Is that something that needs to be documented? Fixed?

Reply:
This is indeed an issue in theory, though in practice, it never really happens due to the checks outside of this call. Not sure how to address this without making a performance regression.

Reply:
It seems like the bad thing is an index out of bounds panic, so maybe just toss in a
```
/// # Panics
/// 
/// Panics if `a` and `b` are equal
```
?